app/utils/algorithm.py

```diff
@@ -4,14 +4,16 @@
 from sqlalchemy import ColumnElement, and_
 from sqlalchemy.orm import Session
 
-from app.models import BlockingAssociation, MatchingPreferences, Profile, ProfileInterestAssociation, User
+from app.models import BlockingAssociation, MatchingPreferences, Profile, ProfileInterestAssociation, RejectedAssociation, User
 
 
 def between(column: int, start: any, end: any) -> ColumnElement[bool]:
     return and_(column >= start, column <= end)
 
 
-def suggest_matches(session: Session, user_id: int = None, number: int = None) -> list[Type[User]]:
+def suggest_matches(session: Session, user_id: int = None, number: int = None, ignore_ids : list[int] = None) -> list[Type[User]]:
+    if ignore_ids is None:
+        ignore_ids = []
     if user_id is None:
         user_id = current_user.id
```

Comment:
Czy tutaj nie możemy tych wartości ustawić jako domyślne?

Reply:
W sumie możemy, odruchowo tak zrobiłem bo w pythonie jest taki śmieszny problem że jak masz domyślny argument mutowalny to jak go zmutujesz to te zmiany się zachowają dla następnych wywołań. Ale tu nie modyfikujemy tej listy więc nie powinno to miec znaczenia.